FakebookNotifications/FakebookNotifications.WebApi/Controller/NotificationController.cs

```diff
@@ -99,17 +102,22 @@ private async Task<IActionResult> CreateNotificationAsync(string loggedInUser, s
             {
                 methodName = "FollowNotification";
             }
+            else
+            {
+                throw new ArgumentException("notificationType is not valid");
```

Comment:
use the paramName argument to this constructor as well

Reply:
and use the switch expression syntax, it's a good fit here

Reply:
"use the paramName argument to this constructor as well"
previous review comment not addressed